### PR TITLE
D1436 grid resize

### PR DESCRIPTION
**Pull request by kyrstenkelly**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (ziroby) [inline index.html]: `index.html` shouldn't be changed if the difference is just in whitespace.

**Comment by ziroby** [comment]
Did you mean to change `gridster.min.js`?  If so, you should put a comment at least in the commit message about why you upgraded.

**Comment by kyrstenkelly** [comment]
I'm not sure why it included the `index.html` in the diff. I did mean to change the minified build, to include the changes I made to the unminified javascript.

**Comment by berhoden** [comment]
LGTM

**Pull request merged**